### PR TITLE
Add galpy

### PR DESCRIPTION
**Pull request by mwcraig**
@jobovy -- please let me know if you are ok with being listed as a maintainer here. The main reason to do so is that when you make a new release you will be able to update the version number/shasum yourself. @bsipocz  and I will help with the conda stuff.

### CODE REVIEW

**Comment by jobovy** [comment]
Wonderful, thanks! Yes, I would be happy to be a maintainer. To update I would change the version number and the hash? Where do I get the hash? I can't this particular one on PyPI.

**Comment by bsipocz** [comment]
Other recipes uses the md5sum, I wonder whether that would be more straightforward here, too? Otherwise I'm also interested in how to get this hash, especially when the release was done by someone else and the only source of the sdist is on pypi.

**Comment by dougalsutherland** [comment]
Command-line way to get the hash:

    curl -SsL https://pypi.io/packages/source/g/galpy/galpy-1.2.tar.gz | openssl sha256

(You might have to `conda install` openssl and/or curl.)

**Comment by dougalsutherland** [comment]
To fix the compile error on Linux/Mac, you might have to set `CPATH` so it can find the gsl headers. Something like:

```yaml
build:
  script: python setup.py install --single-version-externally-managed --record=record.txt  # [win]
  script: CFLAGS="$PREFIX/include" python setup.py install --single-version-externally-managed --record=record.txt  # [unix]
```

**Comment by jobovy** [comment]
@mwcraig This might have to be ``CFLAGS="$PREFIX/include/gsl"``, because the GSL is normally installed in a subdirectory of ``include/`` (this normally fixes this when people have this problem).

**Comment by mwcraig** [comment]
Thanks @jobovy, fixed (hopefully!).

**Comment by dougalsutherland** [comment]
I think what you actually want is `CPATH="$PREFIX/include"`. (I accidentally wrote `CFLAGS` earlier.) It does do `#include <gsl/gsl_spline.h>`, so you don't need the `gsl` part first.

File: recipes/galpy/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record=record.txt  # [win]
-  script: CFLAGS="$PREFIX/include/gsl" python setup.py install --single-version-externally-managed --record=record.txt  # [not win]
+  script: CPATH="$PREFIX/include/gsl" python setup.py install --single-version-externally-managed --record=record.txt  # [not win]
```
Comment (dougalsutherland) [inline recipes/galpy/meta.yaml]: The `/gsl` part is going to break this too. Maybe `CPATH="$PREFIX/include:$CPATH"` would be best.

**Comment by mwcraig** [comment]
I promise, I really can read....I just don't always do it, apparently 😱

**Pull request merged**